pkg/cache/clusterqueue.go

```diff
@@ -343,6 +379,7 @@ func (c *clusterQueue) updateWithAdmissionChecks(checks map[string]AdmissionChec
 	checksPerController := make(map[string][]string, len(c.AdmissionChecks))
```

Comment:
Could you extend the unit testing?
https://github.com/kubernetes-sigs/kueue/blob/5203ac9eb69226a1ae3b6ba6d361eaed7ef3dc8c/pkg/cache/clusterqueue_test.go#L400

Reply:
I would prefer in a follow up. 

I actually tried to extend the unit tests but they seem very unintuitive, because they cover functions without any-user facing impact, so it is hard to interpret them, like: updateClusterQueue, UpdateWithFlavors. So, I thought the integration tests are more reliable here. 

I would prefer unit tests here. My idea was to assert on `inactiveReason` function, and use `updateClusterQueue` for setup.

Reply:
I'm ok with a follow-up.
As I mentioned here (https://github.com/kubernetes-sigs/kueue/pull/3320/files#r1819498096), let's update the spreadsheet.

Reply:
I think this is already covered by the "Add unit test for CQ validation for unsupported configurations (the inactiveReason)" point